services/brig/src/Brig/User/API/Search.hs

```diff
@@ -104,7 +142,34 @@ search searcherId searchTerm maxResults = do
           else do
             -- For team users, we need to check the visibility flag
             Intra.getTeamSearchVisibility t >>= return . handleTeamVisibility t
-  searchIndex searcherId teamSearchInfo searchTerm maxResults
+  Q.searchIndex searcherId teamSearchInfo searchTerm maxResults
   where
     handleTeamVisibility t Team.SearchVisibilityStandard = Search.TeamAndNonMembers t
     handleTeamVisibility t Team.SearchVisibilityNoNameOutsideTeam = Search.TeamOnly t
+
+teamUserSearchH ::
+  ( JSON
+      ::: UserId
+      ::: TeamId
+      ::: Maybe Text
+      ::: Maybe Text
+      ::: Maybe Text
+      ::: Maybe Text
+      ::: Range 1 500 Int32
+  ) ->
+  Handler Response
+teamUserSearchH (_ ::: uid ::: tid ::: mQuery ::: mRoleFilter ::: mSortBy ::: mSortOrder ::: size) = do
+  json <$> teamUserSearch uid tid mQuery (undefined mRoleFilter) mSortBy mSortOrder size
```

Comment:
What is this undefined doing here?

Reply:
It was probably a reminder to not use this argument. It wouldn't cause problems since mRoleFilter, mSortB, mSortOrder are not used, but I should have caught this in my review, and it shouldn't get released like this.